### ISSUE TITLE
Sometimes flood in chat when use tab

### ISSUE THREAD

**Issue by NeadApTar** [corentingosselin/ProdigyNightclubWiki#10]
![image](https://user-images.githubusercontent.com/34768994/121945350-5adc3b80-cd5c-11eb-8683-5162f090ae88.png)

**Comment by corentingosselin** [comment]
Yes this bug is known, it happens when you are outside a nightclub, difficult to fix I am trying to find solutions
